newsletter-july-2021.rst

```diff
@@ -0,0 +1,118 @@
+.. post:: July 8, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from July.
+
+Read the Docs newsletter - July 2021
+====================================
+
+Welcome to a new edition of our monthly newsletter, where we
+openly share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- Eric_ presented the history of the company at the Upstream 2021 conference
+  along with ideas for open source sustainability.
+  You can read more about it in :doc:`our recent blog post </behind-the-scenes-2021-1>`.
+- We revamped our `sustainability page <https://readthedocs.org/sustainability/>`_
+  to clarify the ways in which you or your project can go ad-free.
+- The first two parts of the Sphinx tutorial we are writing
+  in the context of the :doc:`CZI grant </czi-grant-announcement>`
+  were merged to the development branch,
+  and will be publicly available soon.
+- We finalized the design of our upcoming
+  `Embed API v3 <https://docs.readthedocs.io/en/stable/development/design/embed-api.html>`_,
+  which will allow sites outside Read the Docs to embed our documentation.
+- One extra person will join our team as Frontend Developer.
+  More details coming soon!
+
+New features
+------------
+
+- We added a new functionality to rebuild a specific PR build.
+  A small user experience improvement that we hope is useful to many of our users
+  (see image below).
+- We added a JSON schema of our configuration file to
+  `JSON Schema Store <https://www.schemastore.org/json/>`_.
+  If your editor has support for JSON schema,
+  it will now give you autocompletion and highlight for
```

Comment:
```suggestion
  it will now give you autocompletion and linting for
```

Don't know what other term to use, but isn't highlight p:

Reply:
Thanks! Will try to rephrase it

Reply:
"linting" is just fine :)